api-test.c

```diff
@@ -96,6 +96,13 @@ static void async_call_stack_overflow(void)
 
     JSRuntime *rt = JS_NewRuntime();
     JSContext *ctx = JS_NewContext(rt);
+#if !defined(__clang__) || defined(__OPTIMIZE__)
```

Comment:
I don't think this is a good idea. Windows shouldn't crash.

Reply:
reverted.